Procfile.dev

```diff
@@ -0,0 +1,7 @@
+web: unset PORT && bin/rails server -p 4032
```

Comment:
How do you feel about changing this to:
```ruby
bin/rails server -p ${PORT:-4032}
```
To allow individuals to run on a port from their .env file?  e.g. I have mine running on 4050.
This is used in Apply so that adding `PORT=1234` into your .env file runs on 1234, but not having one will run on the default 4032

Reply:
sounds good! However, PORT gets ignored by Procfile. A quick google [revealed this](https://stackoverflow.com/questions/9804184/why-is-foreman-gem-ignoring-the-port-environment-variable)  which works when I rename to SERVER_PORT to avoid conflicts with puma.

Reply:
spoke too soon. Something funny going on and port setting getting ignored!

Reply:
@colinbruce As discussed could not get use of `PORT` to work locally, with odd behaviour where `PORT` is ignored.

Have added a readme amendment commit too.